### PR TITLE
fix(makefile): not building credential helper for windows [EE-4895]

### PR DESCRIPTION
**Pull request by hookenz**
Closes:  [EE-4895](https://portainer.atlassian.net/browse/EE-4895)

[EE-4895]: https://portainer.atlassian.net/browse/EE-4895?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dakota-portainer — APPROVED** [review]
Simple enough, LGTM

File: Makefile [hunk 1]
```diff
@@ -25,7 +25,7 @@ agent: ## Build the agent
 
 credential-helper: ## Build the credential helper (used by edge private registries)
 	@echo "Building Portainer credential-helper..."
-	@cd cmd/$(credential-helper); \
+	@cd cmd/docker-credential-portainer && \
```
Comment (testA113) [inline Makefile:28]: Will windows work with the hardcoded value?

Comment (testA113) [reply Makefile:28]: Got an answer (that `.exe` cause issues with the cd command). LGTM

**Review by testA113 — APPROVED** [review]
LGTM

**Pull request merged**